### PR TITLE
BinaryMesh file format revision 4 (single precision geometry)

### PR DESCRIPTION
**Pull request by dictoon**

### CODE REVIEW

File: src/appleseed/foundation/mesh/binarymeshfilewriter.cpp [hunk 1]
```diff
@@ -48,6 +48,12 @@ namespace foundation
 // BinaryMeshFileWriter class implementation.
 //
 
+namespace
+{
+    // Version of the BinaryMesh file format being written by this code.
+    const uint16 Version = 4;
+}
+
```
Comment (rgirish28) [inline src/appleseed/foundation/mesh/binarymeshfilewriter.cpp]: Not entirely sure why the Version was shifted here, except for readability as no use of Version except in write_version()

Comment (dictoon) [reply src/appleseed/foundation/mesh/binarymeshfilewriter.cpp]: I moved it at the code here just to make it stand out.

Comment (rgirish28) [reply src/appleseed/foundation/mesh/binarymeshfilewriter.cpp]: Cool 👍

**Comment by dictoon** [comment]
Thanks for the review!

**Pull request merged**